src/utils/goto.js

```diff
@@ -22,4 +23,16 @@ const privacyPolicy = () => {
   window.open('https://foundation.m.wikimedia.org/wiki/Privacy_policy')
 }
 
-export const goto = { article, search, consent, termsOfUse, privacyPolicy }
+const randomArticle = (closePopup) => {
```

Comment:
I don't get why this is here. It has a bit too much logic. Functions here should be for simple routing. Interaction with the closePopup functions should stay closer to where that popup is used.

Reply:
Ah I meant to comment about the reasoning here when opening the PR but forgot. It first started with what to do with our `gotToRandomArticle` feature that's so useful for developing. I'm proposing we leave it in the same home page under the CSK [1], thoughts?

Then realized that we need to go to random articles in a couple of the 'Try' tips, so figured we could be more DRY by making that method reusable. Was between a util or as a `goto`, noticed that the name already had 'goto' as a substring so figured why not. Could be a util though.

I agree the `closePopup` feels awkward there though. The reasoning for that is explained by `onTryHandler` in `Tips`: would like to close the popup first before going to a new random article. Let me see if I cook something better though


---

[1]
```
const onKeyCenter = () => {
    const { index, key } = getCurrent()
    if (index) {
      goto.article(lang, key)
    } else if (isRandomEnabled() && !query) {
      goto.randomArticle()
    }
  }
```

Reply:
It can be a function in the search component to keep it dry. Interaction with the closePopup function disqualifies it from being a util.

Reply:
Do you mean like this https://github.com/wikimedia/wikipedia-kaios/pull/313/commits/053d40926109e50ac50d0dc4e1e90465a40ce8ea?